content/1.getting-started/5.voice-chat.md

```diff
@@ -8,4 +8,18 @@ toc: true
 
 # Voice Chat (VC)
 
-:icon{name="noto:building-construction"} Under construction! Author - **Pixel**
\ No newline at end of file
+Fat Duck Gaming requires all players to have a functioning microphone to be able to communicate with other people. The steps below will show how to set up voice chat. 
+
+## Setting Up Voice Chat
+
+1. Press “Escape” and go to settings
+2. Next go to the Voice Chat tab
+3. Make sure your Microphone Enabled is “On”
+4. Under Input Device make sure you select the name of your Microphone and turn your Voice Chat Mode to “Push To Talk”
+5. To setup your Push To Talk bind go to key binds -> GTA Online then Push To Talk 
+
+At any point you are struggling to set up your voice chat please put a /report in the game and contact a staff member to assist. 
+
+![alt text](![Tester](https://github.com/fdg-rp/web-landing/assets/55571792/14d8d7f6-3239-43cc-aeb4-fce865b0d469)
```

Comment:
Incorrect syntax. Should be
`
![alt text](https://github.com/fdg-rp/web-landing/assets/55571792/14d8d7f6-3239-43cc-aeb4-fce865b0d469)
`